spec/models/card_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe Card, type: :model do
+  context 'when validations' do
+    let(:card) { create(:card) }
+
+    it 'when params presence' do
```

Comment:
pls use `when` `should` notation  http://www.betterspecs.org/#short
it means you should write 
```
context 'when something'
  it 'be valid'
    expect(some).to be_valid
  end
end
```

Reply:
Also, good practice to divide unit tests by methods.
Every method is in own `describe` block
in your context it will be
```
describe '#check_translate' do 
  context 'when tranlate is wrong' do 
     it 'returns false' 
        # spec
     end 
  end
end
```